dev/Interop/StoragePickers/PickerLocalization.h

```diff
@@ -5,6 +5,8 @@
 #include "PickerCommon.h"
 
 namespace PickerLocalization {
-    winrt::hstring GetStoragePickersLocalizationText(winrt::hstring key);
+    winrt::hstring GetStoragePickersLocalizationText(winrt::hstring key, winrt::hstring fallback);
```

Comment:
it's still not clear to me why we need a fallback, if we're confident we've fixed root cause of the missing pri.

Reply:
Thank you for your comment, we've removed fallback related code, and currently using a code similar to DeploymentManager